eo-maven-plugin/src/main/java/org/eolang/maven/OyRemote.java

```diff
@@ -81,4 +86,61 @@ public Input get(final String name) throws MalformedURLException {
         return new InputOf(url);
     }
 
+    /**
+     * Hash of head master.
+     * @param hash String "master"
+     * @return SHA of commit
+     * @checkstyle NonStaticMethodCheck (20 lines)
+     */
+    String getSha(final String hash) {
+        String sha = "master";
+        try {
+            final String query = String.format(
+                // @checkstyle LineLength (1 line)
+                "https://api.github.com/repos/objectionary/home/git/refs/heads/%s",
+                hash
+            );
+            final JSONObject obj = readJsonFromUrl(query);
+            sha = obj.getJSONObject("object").getString("sha");
+        } catch (final IOException | JSONException exception) {
+            Logger.info(this, "Couldn't get commit SHA. It will be set as \"master\"");
+        }
+        return sha;
+    }
+
+    /**
+     * Reader.
+     * @param reader Reader
+     * @return String
+     * @throws IOException if something went wrong
+     */
+    private static String readAll(final Reader reader) throws IOException {
```

Comment:
@yasamprom Same here, why is this method `static`?

Reply:
This method doesn't refer anything in this class